app/services/catalog/service_plan_json.rb

```diff
@@ -21,7 +21,7 @@ def process
       relevant_service_plans.each do |plan|
         @reference = plan.portfolio_item.service_offering_ref
         @portfolio_item_id = plan.portfolio_item.id
-        @modified = plan.modified.present?
+        @modified = plan.modified.present? || plan.base.present?
```

Comment:
Isn't `plan.base.present?` always going to be true? We have a `validates :base, :presence => true` on survey plans, and then that makes this `||` just a `true`, right? Seems kinda weird.

Reply:
Oh yeah, that makes sense. Should have probably done that from the beginning, it coincides with what we're doing here: https://github.com/RedHatInsights/catalog-api/blob/64a5b366e7f0e0df63c604368b94d16b10a11b36/app/services/catalog/service_plans.rb#L13